CMSIS/NN/Source/ConvolutionFunctions/arm_depthwise_conv_3x3_s8.c

```diff
@@ -49,71 +49,75 @@
    *  Refer prototype header file for details.
    *
    */
-arm_status arm_depthwise_conv_3x3_s8(const int8_t *input,
-                                     const int32_t input_x,
-                                     const int32_t input_y,
-                                     const int32_t input_ch,
-                                     const int8_t *kernel,
-                                     const int32_t output_ch,
-                                     const int32_t pad_x,
-                                     const int32_t pad_y,
-                                     const int32_t stride_x,
-                                     const int32_t stride_y,
+
+arm_status arm_depthwise_conv_3x3_s8(const cmsis_nn_context *ctx,
+                                     const cmsis_nn_conv_params *conv_params,
```

Comment:
What do you think about having cmsis_nn_dwc_params which includes the depth multiplier as well?

Reply:
This should also help integrating future supports as well

Reply:
Good point. Lets do that!